src/context/worktray-context.tsx

```diff
@@ -267,7 +269,7 @@ export const WorktrayURLProvider: FC<{
   const page = Number(query.get("p") || 1);
   const pageSize = Number(query.get("l") || 10);
   const timePeriod = (query.get("t") || "") as TimePeriodOptions;
-  const patch = query.get("patch") || patchId || "";
```

Comment:
Removing patchId from here breaks the current functionality. If the user has patchId assigned to them then it must be selected when they enter the page for the first time unless they have patch parameter provided in the url already, then the patch from the query will be used.